api-report/container-loader.api.md

```diff
@@ -98,6 +86,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     static createDetached(loader: Loader, codeDetails: IFluidCodeDetails): Promise<Container>;
     // (undocumented)
     get deltaManager(): IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
+    // Warning: (ae-unresolved-inheritdoc-reference) The @inheritDoc reference could not be resolved: The package "@fluidframework/container-loader" does not have an export "DeltaManager"
```

Comment:
Can you go to `packages/loader/container-loader/src/container.ts` and fix the `@inheritDoc` references to either point to the interface or copy them over from `DeltaManager`. As the latter is no longer exported I don't believe our docs can see them anymore...